docs/sphinx/conversion.rst

```diff
@@ -814,3 +814,66 @@ complex type, etc.), even when the logic would be identical for
 e.g. the different integer types.  This approach therefore removes the
 need for unnecessary code duplication, and the type traits checks make
 each type category explicit to the reader.
+
+Behavior differences
+--------------------
+
+Pixel data buffering
+^^^^^^^^^^^^^^^^^^^^
+
+Pixel data is handled differently between the Java and C++
+implementations.  The primary reason for the difference is that the
+Java code uses raw ``byte[]`` arrays to contain pixel data.  This
+could not be implemented in C++ due to the limitations of C++ arrays
+discussed above, as well as having a number of additional limitations:
+
+- they aren't using the native pixel type, requiring conversion to the
+  required type, potentially also including endian conversion
+- they are unstructured, having no dimension ordering or dimension
+  size information
+
+The solution was to create a dedicated :cpp:class:`PixelBuffer`
```

Comment:
This is describing what we did in the past several years back, so I'm not sure the present tense is appropriate.